File: lab4/lab2-Verblud-Javier/punto_3/alcance.java
class Alcance {
	
	void q(int y) {
		System.out.format("Estático pues el valor de y es: %d\n", y);
	}

	void rewrite(int y){

		y = 3;
		q(y);
	}

	public static void main (String args[]) {
		
		int y = 2;
		Alcance a = new Alcance();
		a.rewrite(y);
		System.out.format("Realmente este: %d\n", y);
  }
}

File: lab4/lab2-Verblud-Javier/punto_3/alcance.scala
object Alcance {
	
	def q(y: Int) {
		println("Estático pues el valor de y es: " + y);
	}

	def rewrite(y: Int){
		val y = 3;
		q(y);
	}

	def main(args: Array[String]) {

		// val a = new Alcance()
		val y = 2
		rewrite(y)
		println("Realmente este: " + y)
 	}
}
